pom.xml

```diff
@@ -52,6 +52,18 @@
                         </archive>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>io.github.floverfelt</groupId>
+                    <artifactId>find-and-replace-maven-plugin</artifactId>
+                    <version>1.2.0</version>
+                    <configuration>
+                        <replacementType>file-contents</replacementType>
+                        <fileMask>.md</fileMask>
+                        <!-- Bit of hack as it is assuming this version is always 1.x -->
+                        <findRegex>&lt;version&gt;1.*&lt;/version&gt;</findRegex>
```

Comment:
I'm not entirely happy with this but I think the only other way of doing it would be to remove any `<version>[numeric]...` from the readme that isn't for ide-config ; currently the only other instance is:
```  
  <artifactId>spotless-maven-plugin</artifactId>
  <version>2.44.2</version>
```
Perhaps it would be better to move the above into a properties block so it simplifies the above regex? Thoughts?

Reply:
@rnc Sounds good but my google is failing me. What is the markdown properties block?

Reply:
A properties block like https://github.com/project-ncl/ide-config/blob/main/README.md?plain=1#L50 rather than inlining the version like https://github.com/project-ncl/ide-config/blob/main/README.md?plain=1#L101 which makes the replacement a bit harder.

Reply:
Yeah that makes sense! I was confused about properties, not realizing you meant the pom.xml section in the README! woops

So something like:
```
<properties>
    <ide-config-version>1.0.0</ide-config-version>
    ...
</properties>
<plugin>
  <groupId>net.revelc.code.formatter</groupId>
  <artifactId>formatter-maven-plugin</artifactId>
  <version>${formatter-maven-plugin.version}</version>
  <dependencies>
    <dependency>
      <groupId>org.jboss.pnc</groupId>
      <artifactId>ide-config</artifactId>
      <version>${ide-config-version}</version>
    </dependency>
  </dependencies>
```

and then running the release command will update the Markdown file? If so, LGTM

Reply:
Sort of, just the `spotless-maven-plugin` changed to use a property instead of an inline version and the ide-config being left as `<version>1.0.0</version>`. I've updated the PR to reflect this.

Reply:
Ah right I misunderstood the initial goal. This will allow for only one <version>digit.digit.x</version> for the regex to find and replace, instead of 2.